class TodoNotSavedError(BaseException):
    def __init__(self, msg: str):
        super().__init__(f"Unable to save Todo due to: ${msg}")
